src/gennylib/test/PoolFactory_test.cpp

```diff
@@ -239,6 +239,55 @@ TEST_CASE("PoolFactory behavior") {
         REQUIRE(pool);
     }
 
+    SECTION("Make a pool with client-side encryption enabled") {
```

Comment:
It looks like there are tests for valid combinations of FLE fields in encryption_test.cpp.


Do we need a test with a field with an empty keyId? It looks like this is a valid permutation.